sdk/storage/azure-storage-file-share/tests/perfstress/download.py

```diff
@@ -0,0 +1,36 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from ._test_base import _ShareTest
+
+
+class DownloadTest(_ShareTest):
+    def __init__(self, arguments):
+        super().__init__(arguments)
+        file_name = "downloadtest"
+        self.sharefile_client = self.share_client.get_file_client(file_name)
+        self.async_sharefile_client = self.async_share_client.get_file_client(file_name)
+
+    async def global_setup(self):
+        await super().global_setup()
+        data = b'a' * self.args.size
+        await self.async_sharefile_client.upload_file(data)
+
+    def run_sync(self):
+        stream = self.sharefile_client.download_file(max_concurrency=self.args.parallel)
```

Comment:
You cannot use the `parallel` argument for this purpose, since it's already being used by the framework internally to run `parallel` instances of your test class in parallel.  If you do want an argument for `max_concurrency`, you will need to add a new argument for it, so the user can set `parallel` and `max_concurrency` independently.